### PR TITLE
first functon with pytest

### PR DESCRIPTION
**Pull request by ZarembaBar**

### CODE REVIEW

File: list_comprehension_exercises.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return [number for number in original_list if number % 7 == 0]
+
+
+def all_numbers_form_one_to_thousand_with_three_in_them():
+    original_list = list(range(1, 1001))
+    return [number for number in original_list if "3" in str(number)]
+
+
+def count_the_number_of_spaces_in_string(sentence: str) -> int:
+    return len([space for space in sentence if space == " "])
```
Comment (Eandrju) [inline list_comprehension_exercises.py]: Not every character is a space. Using space as a char name here is misleading.

File: list_comprehension_exercises.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+
+def all_numbers_form_one_to_thousand_divisible_by_seven():
```
Comment (Eandrju) [inline list_comprehension_exercises.py]: Missing output hint type

File: list_comprehension_exercises.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    original_list = list(range(1, 1001))
+    return [number for number in original_list if number % 7 == 0]
+
+
+def all_numbers_form_one_to_thousand_with_three_in_them():
```
Comment (Eandrju) [inline list_comprehension_exercises.py]: The same here - missing hint type

File: list_comprehension_exercises.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    original_list = list(range(1, 1001))
+    return [number for number in original_list if "3" in str(number)]
+
+
+def count_the_number_of_spaces_in_string(sentence: str) -> int:
+    return len([space for space in sentence if space == " "])
+
+
+def list_all_consonants_in_sentence(sentence: str) -> List:
+    return list([letter for letter in sentence if letter not in "aeiouAEIOU "])
```
Comment (Eandrju) [inline list_comprehension_exercises.py:19]: Should there be space in the end?

File: list_comprehension_exercises.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+def all_numbers_form_one_to_thousand_with_three_in_them():
+    original_list = list(range(1, 1001))
+    return [number for number in original_list if "3" in str(number)]
+
+
+def count_the_number_of_spaces_in_string(sentence: str) -> int:
+    return len([space for space in sentence if space == " "])
+
+
+def list_all_consonants_in_sentence(sentence: str) -> List:
```
Comment (Eandrju) [inline list_comprehension_exercises.py]: Missing hint type for lost element. Whenever you use List, there should be a [something]

File: test_list_comprehension_exercises.py [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                                                                     838, 839, 843, 853, 863, 873, 883, 893, 903, 913,
+                                                                     923, 930, 931, 932, 933, 934, 935, 936, 937, 938,
+                                                                     939, 943, 953, 963, 973, 983, 993]
+
+
+def test_count_the_number_of_spaces_in_string_no_space_in_string():
+    assert count_the_number_of_spaces_in_string("BARTEK") == 0
+
+
+def test_count_the_number_of_spaces_in_string_one_space_in_string():
```
Comment (Eandrju) [inline test_list_comprehension_exercises.py:436]: It's always good to write a test for edge cases, like in this example empty string "".

**Review by Eandrju — COMMENTED** [review]
I like it. I like the hint types and tests. And the solutions are correct. One additional task for this session is to run a formatter. Formatter is an external program that modifies your code so that it looks better (removes excessive spaces, break the line in proper way, etc). It DOESN'T change the logic of your program. It just makes it nicer. Thanks to that its easier to read others code, if you follow a style.convention enforced by formatter. It is standard thing in IT to run a formatter, before you push the code. In python the formatter is called "black". Read about it, how to install it and use it. It should be really simple.

File: list_comprehension_exercises.py (hunk 4 above)
Comment (ZarembaBar) [reply list_comprehension_exercises.py:19]: Yes, becouse task was to count only consonants in given sentence. So function was counting consonants and free spaces as different chars. Space at the end, is then like vowel

Comment (Eandrju) [reply list_comprehension_exercises.py:19]: Kk, makes sense

Comment (Eandrju) [reply list_comprehension_exercises.py:19]: What if there are numbers or other characters @#$ then? :*

File: list_comprehension_exercises.py (hunk 5 above)
Comment (ZarembaBar) [reply list_comprehension_exercises.py]: Solved. Char type in List added

File: list_comprehension_exercises.py (hunk 4 above)
Comment (ZarembaBar) [reply list_comprehension_exercises.py:19]: I think error. But in this level of exercise it wasn't case. I have chosen this solution becouse it was faster than writing all consonants small and capital. This would be my solution if exercise would be more complicated.

File: list_comprehension_exercises.py (hunk 2 above)
Comment (ZarembaBar) [reply list_comprehension_exercises.py]: Solved

File: list_comprehension_exercises.py (hunk 3 above)
Comment (ZarembaBar) [reply list_comprehension_exercises.py]: Solved

File: test_list_comprehension_exercises.py (hunk 6 above)
Comment (ZarembaBar) [reply test_list_comprehension_exercises.py:436]: Test added and it is works

File: list_comprehension_exercises.py (hunk 1 above)
Comment (ZarembaBar) [reply list_comprehension_exercises.py]: Solved
